### PR TITLE
Always add trailing '/' to proxyURL

### PR DESCRIPTION
**Pull request by tsnoam**
TL;DR This small patch allows the `client` to contact a remote `proxy` service which runs behind a URL which is not necessarily bound to `/`.

Background:
I've been trying to run PushProx `proxy` behind a reverse proxy.
The reverse proxy was listening on the URL `http://servername/pushprox/`
Once a connection comes in to the reverse proxy, the `servername` is replaced to the internal host and the `/pushprox` is stripped from the URL.
This configuration works just fine from the `proxy` process side. However, the `client` couldn't properly contact the serve because the `/pushprox/` was stripped from the final URL.

The reverse-proxy configuration is based on nginx. This is the configuration block used:
```
        location /promproxy/ {
                # All requests originated to the "/promproxy/" are relayed to the prometheus "proxy" service and "/promproxy/" is truncated from requested URL.
                proxy_pass http://172.17.0.1:8080/;
                # Force HTTP/1.1 so websocket will work (the connection will be kept alive).
                proxy_http_version 1.1;
                proxy_set_header Upgrade $http_upgrade;
        }
```

### CODE REVIEW

**Comment by tsnoam** [comment]
The command line I use:
`client --proxy-url=http://my-server/promproxy`

**Comment by brian-brazil** [comment]
The existing code should already be doing that. Are you sure this is what's going wrong?

**Comment by tsnoam** [comment]
Ok... So I double checked now.

Apparently, if the command is:
`client --proxy-url=http://my-server/promproxy` then the outgoing request will be to `http://my-server/poll`.

However, if the command is ` client --proxy-url=http://my-server/promproxy/` (note the trailing '/') then the outgoing request will be to `http://my-server/promproxy/poll`.

1. I feel embarrassed for not checking this before...
2. I still think that a small fix is needed - the trailing '/' shouldn't be mandatory in input. Would you be interested that I'd provide such a fix?

**Comment by brian-brazil** [comment]
It'd probably be best to always add the trailing /, so the user isn't tripped up by this.

**Comment by tsnoam** [comment]
I've updated the PR title and pushed new code to make sure the proxyURL ends with a single '/'.

**Comment by brian-brazil** [comment]
Thanks!
